### ISSUE TITLE
Enable autodeletion of certain messages

### ISSUE THREAD

**Issue by armand1m** [armand1m/gringobot#1]
**Is your feature request related to a problem? Please describe.**
As people use the bot, the group gets quite flooded with messages from the bot replies. This pollutes the group and the ability to read and navigate the conversation.

**Describe the solution you'd like**
Having the bot auto-deleting the following responses could be potentially helpful:

 - [x] `/estou` response
 - [x] `/sair` response
 - [x] `/ondeestou` response
 
 `/quem` and `/admins` should **not** be deleted.

Ideally, this should be optional. This should be toggleable through an optional environment variable `ENABLE_MESSAGE_AUTO_DELETION` with the default value assigned as `false`.
 
**Describe alternatives you've considered**
Ideally, making the whole process via DM sounds like a good solution, but the same bot instance can run in multiple groups, so the group context is a fairly important implementation detail.

**Comment by AlcidesA** [comment]
Hi, maybe I can help with this issue

I'm kind of new to telegram bots so I'm thinking some ways to achieve this

- get the message_id from replyWithMarkdown return and set a timeout to delete this message
- get chat messages time to time and find these type of messages and delete them (not sure if it's possible)

What do you think ? or if you thought differently

**Comment by armand1m** [comment]
Hey @AlcidesA, I appreciate your eagerness to help! Thank you 😄 

> get the message_id from replyWithMarkdown return and set a timeout to delete this message

I think this would be the ideal approach. We can have a place in the JSON database where we can store the ids for messages that need to be deleted together with the timestamp they've created, and then have a routine job that just goes over these and delete them once their existence exceeds a timeout threshold.

We _could_ do it in memory but in case of a service failure that could be lost. Having it stored in the JSON database will be handy since we can run this job routine during startup as well, so if the container crashes, it will still be able to delete a message that should be deleted at some point.

**Comment by AlcidesA** [comment]
Anytime!! 

Ok, sounds good to me, when you say routine job, you mean something like a cron job ? 

I'll start to work on this :punch:

**Comment by armand1m** [comment]
@AlcidesA 

more like an interval running on the process itself, something as simple as this:

```ts
const runMessageRecycling = () => {
  /** logic related to recycling messages scheduled to deletion in the database*/
};

const interval = setInterval(runMessageRecycling, 60000)
```

**Comment by armand1m** [comment]
Feature was implemented and merged at #3

Container image with the feature is deployed at SHA https://github.com/armand1m/gringobot/commit/c4fa3b490208eeafa63b6654cedfed62fa4730a9
